docs/index.html

```diff
@@ -3,9 +3,13 @@
 <head>
   <meta charset="utf-8" />
   <title>PokéSprite</title>
+  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
   <link type="text/css" href="build/pokesprite.min.css" rel="stylesheet" media="screen" />
-  <script charset="utf-8" src="build/pokesprite.min.js" ></script>
   <link type="text/css" href="build/overview.min.css" rel="stylesheet" media="screen" />
+  <style>
+     button {padding:5px 15px;background:#000;border:0 none;cursor:pointer;-webkit-border-radius: 5px;border-radius: 5px;color: #fff;font-size:14px;line-height: 21px;} #button-center {text-align: center;}.fa{font-size:21px;margin-right:4px;}
+  </style>
+  <script charset="utf-8" src="build/pokesprite.min.js" ></script>

```

Comment:
There's an extra space here. Seems I added that. You could fix that while we're at it.